ts/src/bitget.ts

```diff
@@ -2753,11 +2755,11 @@ export default class bitget extends Exchange {
             'orderType': type,
         };
         const isMarketOrder = type === 'market';
```

Comment:
@Dan-krm right now we have this on line 2769
```Javascript
        const isStopLossOrTakeProfit = isStopLoss || isTakeProfit;
        if (this.sum (isTriggerOrder, isStopLossTriggerOrder, isTakeProfitTriggerOrder, isStopLoss, isTakeProfit) > 1) {
            throw new ExchangeError (this.id + ' createOrder() params can only contain one of triggerPrice, stopLossPrice, takeProfitPrice, stopLoss, takeProfit');
        }
```

but it does not seem correct, can't you have a takeProfit+stopLoss simultaneously?

Reply:
Yeah good call, I'll push some changes